src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -2235,7 +2236,7 @@ private void WatchForGitChanges()
 				var notifyFilters = FILE_NOTIFY_CHANGE_DIR_NAME | FILE_NOTIFY_CHANGE_FILE_NAME | FILE_NOTIFY_CHANGE_LAST_WRITE | FILE_NOTIFY_CHANGE_SIZE | FILE_NOTIFY_CHANGE_CREATION;
 
 				var overlapped = new OVERLAPPED();
-				overlapped.hEvent = CreateEvent(IntPtr.Zero, false, false, null);
+				overlapped.hEvent = PInvoke.CreateEvent(null, false, false, null).DangerousGetHandle();
```

Comment:
Asa above.

Reply:
@0x5bfa what do you want me to change here and in the above review?
The OVERLAPPED legacy struct only allows setting IntPtr as the hEvent property.

I replaced this with the `System.Runtime.InteropServices.Overlapped` type in #17108.